### PR TITLE
README: add slack to README and spruce things up

### PR DESCRIPTION
**Pull request by jess-edwards**
Adds slack channel under the Support section in the read.me

Can someone help me update the logo? I'm not sure where to upload it.

Release justification: Non-production change.
Release note: None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
   Learn how to troubleshoot common errors, cluster and node setup, and SQL query behavior,
   and how to use debug and error logs.
 
+- #[CockroachDB Community Slack](https://cockroa.ch/slack)
```
Comment (irfansharif) [inline README.md]: Remove the "#" and the new line below.

**Review by irfansharif — APPROVED** [review]
LGTM. If by logo you mean the top level cockroachdb one, you're looking for `docs/media/cockroach_db.png`.

**Comment by jess-edwards** [comment]
@irfansharif could you help me make these changes and merge the PR? I don't have permissions.

**Review by rafiss — APPROVED** [review]
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46245)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @jess-edwards, @jordanlewis, and @rafiss)

**Review by rafiss — APPROVED** [review]
lgtm

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46245#-:-M3MGMsf5ugDavhE6Lqq:bb7k0qh)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @jess-edwards, @jordanlewis, and @rafiss)

**Comment by irfansharif** [comment]
bors retry

**Comment by jess-edwards** [comment]
oh man. bors says no today.

**Comment by irfansharif** [comment]
bors retry

**Pull request merged**